interface/patient_file/encounter/Services/ViewHelper.php

```diff
@@ -0,0 +1,264 @@
+<?php
+/**
+ * Helper functions for an encounter view files
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License as published by the Free Software Foundation; either version 3 of the License, or (at your option)
+ * any later version. This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+ * License for more details. You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @subpackage Encounter
+ * @license GNU GPL3
```

Comment:
sorry for the numerous separate posts on this. I completely forgot to mention, that also need to have full headers if do things like combining code from 2 separate licenses, such as here: 
https://www.mozilla.org/en-US/MPL/2.0/combining-mpl-and-gpl/